### DISCUSSION TITLE
fix file & folder stuktur

### DISCUSSION THREAD

**Discussion by ENONGLOSKER** [ENONGLOSKER/ONE-REPO#6] [Q&A]
masukan ke dalam folder img

@samsuljihad 
![ji](https://github.com/ENONGLOSKER/ONE-REPO/assets/75899483/b6d9f194-c431-4d6b-8335-cb9d9c6ec123)

**Comment by samsuljihad** [comment]
sorry gaes itu tidak di pakai,sudah ada di file img


Pada Sel, 11 Jun 2024 22.00, ELQUSAIRI ***@***.***> menulis:

**Reply by ENONGLOSKER** [reply]
di hapus klo memang tidak dipake

**Reply by samsuljihad** [reply]
udah el

**Comment by samsuljihad** [comment]
skrg tak hapus

Pada Sel, 11 Jun 2024 22.40, samsul jihad ***@***.***>
menulis:

**Reply by ENONGLOSKER** [reply]
oke dh
